modules/cli/src/main/scala/scala/cli/commands/export0/ExportOptions.scala

```diff
@@ -27,6 +27,11 @@ final case class ExportOptions(
   @HelpMessage("Sets the export format to SBT")
     sbt: Option[Boolean] = None,
   @Group(HelpGroup.BuildToolExport.toString)
+  @Tag(tags.experimental)
+  @Tag(tags.inShortHelp)
+  @HelpMessage("Sets the export format to Maven")
```

Comment:
`--mvn` as an alias seems reasonable
```suggestion
  @Name("mvn")
  @HelpMessage("Sets the export format to Maven")
```

Reply:
ah... I meant to add `mvn` as an alias, while keeping `maven`. see the suggested code.